src/App.jsx

```diff
@@ -1,17 +1,74 @@
 import { initialColors } from "./lib/colors";
 import Color from "./Components/Color/Color";
+import ColorForm from "./Components/ColorForm/ColorForm";
 import "./App.css";
+import { useState } from "react";
+import { uid } from "uid";
+// DONE FOR NOW
 
+console.log("App.jsx is connected!");
+
+// onSubmitColor muss von hier mitgegeben werden!
 function App() {
+  // von JEssica:
+  const [colors, setColors] = useState(initialColors);
+  console.log("allColors", colors);
+
+  // Hier definieren wir jetzt die Funktion OnSubmit
```

Comment:
Hey Rebecca,

I'm a bit confused by the comments you included in your code. I used to do that as well, but now I save my comments elsewhere to keep my code clean and more readable. Perhaps in our next project, we can improve our code's clarity together. This is also a helpful review for me too, hehe. But so far it is good and function work as well :))